noxfile.py

```diff
@@ -1109,7 +1109,15 @@ def test_introspection(session: nox.Session):
     profile = os.environ.get("CARGO_BUILD_PROFILE")
     if profile == "release":
         options.append("--release")
-    session.run_always("maturin", "develop", "-m", "./pytests/Cargo.toml", *options)
+    session.run_always(
+        "maturin",
+        "develop",
+        "-m",
+        "./pytests/Cargo.toml",
+        "--features",
+        "experimental-inspect",
```

Comment:
I kept the change in `pytests/Cargo.toml` that makes optional the `experimental-inspect` feature